scripts/abby_excel.py

```
@@ -0,0 +1,183 @@
+    else:
+        return sum(data) / len(data)
+
+
+# Get the current working directory
+current_dir = os.getcwd()
+
+# Add it to sys.path
+if current_dir not in sys.path:
+    sys.path.append(current_dir)
+
+from src.jsonsync import JsonItem
```

Ruff (0.12.2)

Module level import not at top of file

(E402)